pkg/v5/configmap/current.go

```diff
@@ -0,0 +1,42 @@
+package configmap
+
+import (
+	"context"
+
+	"github.com/giantswarm/microerror"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (s *Service) GetCurrentState(ctx context.Context, config ConfigMapConfig) ([]*corev1.ConfigMap, error) {
+	var currentConfigMaps []*corev1.ConfigMap
+
+	guestK8sClient, err := s.guest.NewK8sClient(ctx, config.ClusterID, config.GuestAPIDomain)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
+	// Namespaces used by all providers. Uses a map for deduping.
```

Comment:
Getting slangy here eh mate?

Reply:
Jeje yes I tend to use more slang on Fridays! ❤️